Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridDataConnection.cs

```diff
@@ -197,6 +202,22 @@ public Type DataType
             }
         }
 
+        public bool IsDataSourceIncremental
```

Comment:
Public properties should remain in alphabetical order within the public properties.